### ISSUE TITLE
Shift+enter to create new line in same paragraph

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/web-components#182]
# Feature Request 🛍️
currently pressing enter creates a new paragraph, however there is no way to create a new line in the same paragraph. In most document editing software, shift+enter creates a new line within the same paragraph. Let's bring this functionality to AP so that we can provide a more consistent experience for people used to common document editors like Microsoft Word and Google Docs.

**Comment by dselman** [comment]
Editor support for SHIFT-ENTER to insert a markdown linebreak (`<br/>` in HTML) has been added. Closing this - please create new issues if this is insufficient with additional details.
